core/chain_manager.go

```diff
@@ -48,6 +48,9 @@ var (
 )
 
 const (
+	headerCacheLimit    = 512
+	bodyCacheLimit      = 256
+	tdCacheLimit        = 1024
```

Comment:
Olympic has several periods of intense stress testing / spam - we could extract some of this for use in benchmarks.